dumpling/export/dump.go

```diff
@@ -32,6 +33,7 @@ import (
 	"github.com/pingcap/tidb/store/helper"
 	"github.com/pingcap/tidb/tablecodec"
 	"github.com/pingcap/tidb/util/codec"
+	"github.com/pingcap/tiflow/pkg/container/queue"
```

Comment:
tidb should not import tiflow to avoid import cycle. You can copy the needed data structure here